testbed/core/models.py

```diff
@@ -363,6 +363,92 @@ def __str__(self):
         return f'{username} follows {self.actor.username} (remote)'
 
 
+class Blocked(models.Model):
+    """
+    LOLA Blocked collection model representing current blocking state.
```

Comment:
I don't think this is a collection model...  It's just one entry in one actor's blocked collection, right